python/az/aro/azext_aro/custom.py

```diff
@@ -25,6 +25,7 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
                vnet=None,
                vnet_resource_group_name=None,  # pylint: disable=unused-argument
                location=None,
+               pull_secret=None,
```

Comment:
If this defaulted to `''` here, then you wouldn't need to short-circuit on line 72.

Reply:
I don't know if defaulting to None is a requirement of the az code which wraps us.